test/Twig/Tests/EnvironmentTest.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-require_once dirname(__FILE__).'/FilesystemHelper.php';
+require_once __DIR__.'/FilesystemHelper.php';
```

Comment:
Should be reverted. Even if you're right here, I prefer to not upgrade the code to 5.3+ in 1.x. It has been done in 2.x and I don't want people starting to send random PR about such changes :)